.travis.yml

```diff
@@ -2,9 +2,21 @@ dist: bionic
 language: python
 python:
   - "3.7"
-install:
-  - pip install pylint
 jobs:
   include:
     - name: pylint
-      script: pylint osbuild assemblers/* stages/*
+      install: pip install pylint
+      script: pylint osbuild osbuild-run assemblers/* stages/*
+    - name: rpm
+      before_install:
```

Comment:
Travis docs are abysmal. Most of the stuff I do I just made up, and it "just worked". No idea about the reason for the distinction, just by cargo-culting, my take is that install is for pip (since we are on python) and before_install for "other things". But it really makes no sense. Those guys should write docs. Any docs.